Documentation/physical-models.rst

```diff
@@ -5,6 +5,9 @@ Overview
 --------
 
 SeisSol includes various physical models to simulate realistic earthquake scenarios.
+SeisSol assumes constant material values per element.
+By default, the material values are sampled at the element barycenter. For (visco)elastic materials though, material values can be sampled by averaging over the whole element (c.f. https://mediatum.ub.tum.de/node?id=1664043). 
```

Comment:
By default, the material values are sampled at the element barycenter.
For elastic (:math:`\rho`, :math:`\mu`, and :math:`\lambda`) and viscoelastic (:math:`\rho`, :math:`\mu`, :math:`\lambda`, :math:`Q_p` and :math:`Q_s`) materials though, cell-homogenized (averaged) properties can be computed and used (c.f. https://mediatum.ub.tum.de/node?id=1664043).